README.md

```diff
@@ -0,0 +1,156 @@
+
+```
+./build.py -v  --enable-logging --enable-stats --enable-tracing
+                --enable-metrics --enable-gpu-metrics --enable-cpu-metrics
+                --enable-gpu
+                --filesystem=gcs --filesystem=s3 --filesystem=azure_storage
+                --endpoint=http --endpoint=grpc --endpoint=sagemaker --endpoint=vertex-ai
+                --backend=python:r23.10
+                --backend=vllm:r23.10
+```
+
+### Option 3. Add the vLLM Backend to the Triton Container
+
+You can install the vLLM backend directly into the NGC Triton container.
+In this case, please install vLLM first. You can do so by running
+`pip install vllm==<vLLM_version>`. Then, set up the vLLM backend in the
+container with the following commands:
+
+```
+mkdir -p /opt/tritonserver/backends/vllm
+wget -P /opt/tritonserver/backends/vllm https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/src/model.py
+```
+
+## Using the vLLM Backend
+
+You can see an example
+[model_repository](samples/model_repository)
+in the [samples](samples) folder.
+You can use this as is and change the model by changing the `model` value in `model.json`.
+`model.json` represents a key-value dictionary that is fed to vLLM's AsyncLLMEngine when initializing the model.
+You can see supported arguments in vLLM's
+[arg_utils.py](https://github.com/vllm-project/vllm/blob/main/vllm/engine/arg_utils.py).
+Specifically,
+[here](https://github.com/vllm-project/vllm/blob/ee8217e5bee5860469204ee57077a91138c9af02/vllm/engine/arg_utils.py#L11)
+and
+[here](https://github.com/vllm-project/vllm/blob/ee8217e5bee5860469204ee57077a91138c9af02/vllm/engine/arg_utils.py#L201).
+
+In the [samples](samples) folder, you can also find a sample client,
+[client.py](samples/client.py).
+
+## Running the Latest vLLM Version
+
+To see the version of vLLM in the container, see the
+[version_map](https://github.com/triton-inference-server/server/blob/85487a1e15438ccb9592b58e308a3f78724fa483/build.py#L83)
+in [build.py](https://github.com/triton-inference-server/server/blob/main/build.py)
+for the Triton version you are using. These are compatible with the newer versions of CUDA running in Triton.
+
+If you would like to use a specific vLLM commit or the latest version of vLLM, you
+will need to use a
+[custom execution environment](https://github.com/triton-inference-server/python_backend#creating-custom-execution-environments).
+
+
+## Sending Your First Inference
+
+After you
+[start Triton](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/getting_started/quickstart.html)
+with the
+[sample model_repository](samples/model_repository),
+you can quickly run your first inference request with the
+[generate endpoint](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_generate.md).
+
+Try out the command below.
+You can replace _client input_ with your input text.
```

Comment:
client_input is rendering quite differently. Can you fix it?

Reply:
I'll take a look

Reply:
This was intentional to show that you need to replace "client input" in the command. Would you prefer a different formatting?

Reply:
Used quotes instead to be clearer.